File: JAVASCRIPT/CreditCard.js
const prompt = require("prompt-sync")(); 

let cardNumber = prompt("Enter your credit card number: ");

if (cardNumber.length < 13 || cardNumber.length > 16) {
    console.log("Invalid credit card number!");
    process.exit();
}

let cardDigits = [];
for (let i = 0; i < cardNumber.length; i++) {
    cardDigits.push(parseInt(cardNumber.charAt(i)));
}

if (cardDigits[0] === 4) {
    console.log("\n<<<< credit card type: VISA CARD");
} else if (cardDigits[0] === 5) {
    console.log("\n<<<< credit card type: MASTERCARD");
} else if (cardDigits[0] === 3 && cardDigits[1] === 7) {
    console.log("\n<<<< credit card type: AMERICAN EXPRESS CARD");
} else if (cardDigits[0] === 6) {
    console.log("\n<<<< credit card type: DISCOVER CARD");
} else {
    console.log("CARD NOT RECOGNIZED!");
}

console.log("<<<< credit card Number: " + cardNumber);
console.log("<<<< credit card Digit: " + cardNumber.length);

File: JAVA/CreditCard.java
import java.util.Scanner;
public class CreditCard {

public static void main(String[] args) {

Scanner input = new Scanner(System.in);


System.out.println("Enter your credit card number");
String cardNumber = input.nextLine();




if(cardNumber.length() < 13 || cardNumber.length() > 16) {


System.out.print("Invalid credit card number!");

return;


}

int[] cardDigits = new int[cardNumber.length()];

for(int count = 0; count < cardNumber.length(); count++) {


cardDigits[count] = Character.getNumericValue(cardNumber.charAt(count));

}




if (cardDigits[0] == 4) {
System.out.println("\n<<<< credit card type: VISA CARD");
} else if (cardDigits[0] == 5) {
System.out.println("\n<<<< credit card type: MASTERCARD");
} else if (cardDigits[0] == 3 && cardDigits[1] == 7) {
System.out.println("\n<<<< credit type : AMERICAN EXPRESS CARD");
} else if (cardDigits[0] == 6) {
System.out.println("\n<<<< credit card type: DISCOVER CARD");
} else {
System.out.println("CARD NOT RECOGNIZED!");
}

System.out.println("<<<< credit card Number: " + cardNumber);
System.out.println("<<<< credit card Digit: " + cardNumber.length());






















} // class


} // main